docs/index.md

```diff
@@ -47,7 +47,7 @@ Foam is a like a bathtub: _What you get out of it depends on what you put into i
 
 Like the soapy suds it's named after, **Foam** is mostly air.
 
-1. The editing experience of **Foam** is powered by VS Code, enhanced by workspace settings that glue together [[recommended-extensions]] and preferences optimised for writing and navigating information.
+1. The editing experience of **Foam** is powered by VS Code, enhanced by workspace settings that glue it together [[recommended-extensions]] and preferences optimised for writing and navigating information.
```

Comment:
In this change the original is as intended. Workspace settings glue recommended extensions and preferences together, but with inverted word order "glue together". Not sure if this is grammatically correct, but the suggested change is definitely not.

Roll this one back?

Reply:
ok, I see where I misread,  the linked doc caused me to read it like a pause/comma was there.

Reply:
I have reverted this change.